contents/install/new_pod/index.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Install diaspora*
+nav_title: Installation
+subtitle: Build your own home.
+---
+
+**This is a guide for server administrators or developers**, and while we do not want to scare you away, we mean that. If you want to set up a production-ready diaspora\* on a server, you should be a *somewhat* experienced server admin. There is only so much we can cover in this guides, and we will not get into setting up the server itself. This means that things like setting up firewalls, performing regular updates, and doing other things that keep your setup secure is beyond our scope, and you should be very comfortable taking care of that yourself. There is nothing worse than using diaspora\*, but having your data, or even worse - other people's data, leaked, because you ran an outdated OpenSSH version.
+
+**Following these guides will get you the most help, if you need it**. diaspora\* is a complex application, and so can be debugging it. These guides are written and maintained by the project's development team and result in setups that we understand, and can support in case something goes wrong. We are aware that there are other guides out there, and while we cannot stop you from giving those a try, you might find it hard to get help if you run into unexpected troubles.
+
+**The setup is a bit complex, but we are here to help**. We know it can be overwhelming, but [we are here to help out][get-help] if you need us. If something goes wrong, or you have additional questions, please do not hesitate to reach out!
```

Comment:
```suggestion
**The setup is a bit complex, but we are here to help**. We know it can be overwhelming, so don't hesitate to [ask for help][get-help] if you need us. If something goes wrong, or you have additional questions, please do reach out!
```